meta-balena-common/recipes-support/resin-mounts/resin-mounts.inc

```diff
@@ -55,6 +55,9 @@ do_compile () {
 			# This bind mount is only invoked manually in persistent logs
 			# service. No need to run it with default target.
 			sed -i -e "/^WantedBy=/ d" $servicefile
+			# Mount persistent logs on data partition
+			sed -i -e 's/resin-state.service resin-state-reset.service/resin-data.service/g' $servicefile
+			sed -i -e 's#/mnt/state/root-overlay#/mnt/data/resin-data/logs#g' $servicefile
```

Comment:
So is the full path of the logs on the disk becomes `/mnt/data/resin-data/logs/var/log/journal`?
That feels weird.. 
How about `/mnt/data/root-overlay` in case we want to move other things from the state partition later on as well?